tools/ci/watchdog.sh

```diff
@@ -38,8 +38,26 @@ CMD_EXIT="/tmp/watchdog.exit"
 # Utility functions
 # ============================================= 
 
+max_of() {
+  local max number
+
+  max="$1"
+
+  for number in "${@:2}"; do
+    if ((number > max)); then
+      max="$number"
+    fi
+  done
+
+  printf '%d\n' "$max"
```

Comment:
I forgot to ask here: Is there a reason why you didn't just use `echo "$max"` instead?

Reply:
Well, my code is (of course) inspired by Stackoverflow. 
But there seems to be some arguments why printf is better than echo: https://unix.stackexchange.com/questions/65803/why-is-printf-better-than-echo

Reply:
:-D I like how the accepted answer feels to be 10 pages long for something I considered a minor thing ^^